src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5438,6 +5440,13 @@ public override IEnumerable<IOperation> Children
                 {
                     yield return Initializer;
                 }
+                if (!IgnoredArguments.IsEmpty)
```

Comment:
>if (!IgnoredArguments.IsEmpty) [](start = 16, length = 30)

I would return them before the Initializer because syntactically they come first.

Reply:
Done.

---
In reply to: [149214465](https://github.com/dotnet/roslyn/pull/23008#discussion_r149214465) [](ancestors = 149214465)